tools/src/main/scala/caliban/tools/Codegen.scala

```diff
@@ -2,24 +2,38 @@ package caliban.tools
 
 import java.io.{ File, PrintWriter }
 
-import caliban.parsing.adt.Document
+import caliban.tools.implicits.ScalarMappings
 import zio.{ Task, UIO }
 
 object Codegen {
 
+  sealed trait GenType
+
+  object Schema extends GenType
```

Comment:
I usually put these inside `object GenType` so that we can call them `GenType.Schema`